subsys/testsuite/ztest/include/ztest_test_new.h

```diff
@@ -210,6 +210,17 @@ static inline void unit_test_noop(void)
 #define Z_ZTEST(suite, fn, t_options) Z_TEST(suite, fn, t_options, 0)
 #define Z_ZTEST_F(suite, fn, t_options) Z_TEST(suite, fn, t_options, 1)
 
+/**
+ * @brief Skips the test if config is enabled
+ *
+ * Use this macro at the start of your test case, to skip it when
+ * config is enabled.  Useful when your test is still under development.
+ *
+ * @param config The Kconfig option or statically assertable statement used to
+ *		 skip the test.
+ */
+#define Z_TEST_SKIP_IFDEF(config) COND_CODE_1(config, (ztest_test_skip()), ())
```

Comment:
I'm sorry to do this again. `(config)` to avoid parentheses issues in macro expansion